stories/topics/tech-note-gcp-load-balancer-playbooks.adoc

```diff
@@ -0,0 +1,10 @@
+[id="tech-note-gcp-load-balancer-playbooks"]
+
+= GCP External load balancer playbooks are not supported
+
+{AAPonGCP} does not support the use of the following playbooks:
+
+* gcp_create_external_load_balancer
+* gcp_delete_external_load_balancer
+
+These playbooks are not guaranteed to work and are not supported.
```

Comment:
Should these two be removed from the list of available playbooks until they are supported? Just a thought.
Otherwise
LGTM

Reply:
Good idea.  Even though the user might be able to see it currently, we should remove from the listing and only have the tech note.